core/tracepoint/path_spec.rb

```diff
@@ -0,0 +1,11 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+ruby_version_is '2.0' do
+  describe 'TracePoint#path' do
```

Comment:
Now that I think of it, we should spec what happens when the code running is not from a file (e.g. ran using `eval`)

Reply:
yes, I have added the spec.